Description: Debianize
 Do not use embedded version, but the one of the libjs-lrdragndrop.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-03-23

--- python-xstatic-angular-lrdragndrop-1.0.2.1.orig/xstatic/pkg/angular_lrdragndrop/__init__.py
+++ python-xstatic-angular-lrdragndrop-1.0.2.1/xstatic/pkg/angular_lrdragndrop/__init__.py
@@ -38,9 +38,9 @@ HOMEPAGE = 'https://github.com/lorenzofo
 LICENSE = '(same as %s)' % DISPLAY_NAME
 
 from os.path import join, dirname
-BASE_DIR = join(dirname(__file__), 'data')
+#BASE_DIR = join(dirname(__file__), 'data')
 # linux package maintainers just can point to their file locations like this:
-#BASE_DIR = '/usr/share/javascript/' + NAME
+BASE_DIR = '/usr/share/javascript/lrDragNDrop'
 
 # location of the Javascript file that's the entry point for this package, if
 # one exists, relative to BASE_DIR
